client/api.py

```diff
@@ -14,17 +12,17 @@
 @app.route('/website', methods=['GET'])
 def get_metadata():
     url = request.args.get('src')
-    response =  url_utils.get_url_data(url)
-    # TODO(Meenu): Scraper must take in a response object and not a url
-    connection = requests.get(response.url)
-    if "wikipedia" in response.url:
+    response = url_utils.get_url_data(url)
+    scraper = get_scraper(response)
```

Comment:
In the future, this api call should just call some other function in a different file and let it handle all the logic